client/views/changelog.tpl

```diff
@@ -0,0 +1,27 @@
+{{#if latest}}
+	<h3>New version available!</h3>
+	<p>
+		There is a new {{#if latest.prerelease}}pre-{{/if}}release <b>{{latest.version}}</b> of The Lounge available.
+		<a href="{{latest.url}}" target="_blank" rel="noopener">View details on GitHub</a>
+	</p>
+	<p>We strongly suggest updating The Lounge to the latest version.</p>
```

Comment:
I would remove "The Lounge".

`<p>We strongly suggest updating to the latest version.</p>`